datasist/datasist/structdata.py

```diff
@@ -75,9 +75,16 @@ def describe(data=None, name='', date_cols=None, show_categories=False, plot_mis
     display(cat_features)
     _space()
     
+    print("\n--------Null Value------")
+    display(data.isna().sum())
+          
     print('Unique class Count of Categorical features')
     display(get_unique_counts(data))
     _space()
+    
+    print('Value count the no. of non-NA/null observations across the given axis. It works with non-floating type data as well.)

```

Comment:
Since we have printed the missing values count, displaying the not-missing seems counter-intuitive